modules/camera/camera_osx.mm

```diff
@@ -313,20 +313,31 @@ - (void)dealloc {
 // CameraOSX - Subclass for our camera server on OSX
 
 void CameraOSX::update_feeds() {
+#if MAC_OS_X_VERSION_MIN_REQUIRED >= 101500
+	AVCaptureDeviceDiscoverySession *session = [AVCaptureDeviceDiscoverySession discoverySessionWithDeviceTypes:[NSArray arrayWithObjects:AVCaptureDeviceTypeExternalUnknown, AVCaptureDeviceTypeBuiltInWideAngleCamera, nil] mediaType:AVMediaTypeVideo position:AVCaptureDevicePositionUnspecified];
```

Comment:
Assuming that the data structure is the same, I guess you could add `NSArray *devices = session.devices;` here and avoid the other `#if` checks below.